src/main/java/teammates/storage/sqlentity/FeedbackSessionLog.java

```diff
@@ -27,7 +29,7 @@ public class FeedbackSessionLog extends BaseEntity {
     private String feedbackSessionName;
 
     @Column(nullable = false)
```

Comment:
we should add: @Enumerated(EnumType.STRING) below as well (see Instructor entity for example)